src/app/[locale]/(app)/credits/_components/Contributors.tsx

```diff
@@ -0,0 +1,155 @@
+"use client";
+
+import Section from "~/components/Section";
+import contributorsData from "../../../../../../contributors.json";
```

Comment:
Will it be a good idea to add this to import alias?

Reply:
I have the same issue with package.json in `src/constants.ts`. Let keep this like this for now but could you open a PR for this?